### DISCUSSION TITLE
Does this work with latest version of OSX?

### DISCUSSION THREAD

**Discussion by mastoj** [AdamWagner/stackline#121] [Q&A]
I just got my Mac after being on Linux for a while and found this little nugget. My only problem is that I can't get it to work. Yabai works and stacking works, but no indicator.

**Comment by stvsu** [comment]
Also curious about this after updating to Sonoma. Coming from sway on Linux and heavily utilizing the tabbing functionality; this is the "final" piece missing from yabai to be a full replacement for me.

**Comment by Zhanweelee** [comment]
Seems like it's not working with Sonoma and Latest yabai anymore?
